addons/point_of_sale/static/tests/tours/utils/product_screen_util.js

```diff
@@ -376,7 +376,7 @@ export function modeIsActive(mode) {
 export function noDiscountApplied(originalPrice) {
     return inLeftSide({
         content: "no discount is applied",
-        trigger: `.orderline .info-list:not(:contains(${originalPrice}))`,
+        trigger: `.orderline .info-list:contains(${originalPrice})`,
```

Comment:
i don't understand this change, can you explain? why do we suddenly don't need the "not".

Reply:
Hello @caburj 
It seems we don't need this method either, as we're not altering the orderline amount or the total amount when the 
tax is inclusive.

Reply:
@caburj can you please check again?
Thanks!